.pipelines/versionAndSignCheck.ps1

```diff
@@ -7,7 +7,7 @@ Param(
 )
 
 $DirPath = $targetDir;  #this file is in pipeline, we need root.
-$items = Get-ChildItem -Path $DirPath -File -Include *.exe,*.dll -Recurse -Force -ErrorAction SilentlyContinue
+$items = Get-ChildItem -Path $DirPath -File -Include *.exe,*.dll,*.ttf -Recurse -Force -ErrorAction SilentlyContinue
```

Comment:
Do ttf files need to be signed?

Reply:
@crutkas said yes. He will look into it in a future PR

Reply:
Should they be OTF instead?